### PR TITLE
[Feat] : User profile, discover new people

### PR DESCRIPTION
**Pull request by Logan1x**
This PR includes a mix of three features,
- User profile feature
- individual user profile by username
- follow/ discover new people
- Post routes and API is being setup

### CODE REVIEW

File: src/features/postSlice.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        headers: {
+          authorization: authToken,
+        },
+        data: JSON.stringify({
+          postData: post,
+        }),
+      });
+      return res.data;
+    } catch (err) {
+      console.log(err);
```
Comment (manish591) [inline src/features/postSlice.js:39]: use console.error

File: src/pages/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                type="text"
+                value={updateUserData.lastName}
+                onChange={(e) =>
+                  setUpdateUserData({
+                    ...updateUserData,
+                    lastName: e.target.value,
+                  })
+                }
+              />
+              <label htmlFor="">Username</label>
```
Comment (manish591) [inline src/pages/Profile.jsx:102]: Don't keep htmlfor empty

**Review by manish591 — APPROVED** [review]
Code is looking good.

**Comment by Logan1x** [comment]
Both of the changes are solved in future commits. Thanks for pointing it out :)

**Review by MonikaJhingan — APPROVED** [review]
The vode looks good to be merged.

**Pull request merged**